cmd/livepeer_router/livepeer_router.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"flag"
+	"net/url"
+	"os"
+	"os/signal"
+	"os/user"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/server"
+)
+
+func defaultAddr(addr, defaultHost, defaultPort string) string {
```

Comment:
I'm okay with duplicating this, alternatively could move it to `common.DefaultAddr` as it's also used in `cmd/livepeer.go`